### PR TITLE
ATTACHMENT FLAVOR TEXT GO

### PR DESCRIPTION
**Pull request by KoishiVibe**
# Summary

new year new flavor text
also includes UPP variants for most of the attachments (cosmetic)

# Explain why it's good for the game

muh immersion

### CODE REVIEW

**Comment by KoishiVibe** [comment]
it should be ready technically.

File: code/modules/projectiles/gun_attachables.dm [hunk 1]
```diff
@@ -3197,6 +3215,10 @@ Defined in conflicts.dm of the #defines folder.
 	slot = "under"
 	pixel_shift_x = 20
 
+/obj/item/attachable/angledgrip/union
+	name = "RK-9 Vertical Grip"
```
Comment (AndroBetel) [inline code/modules/projectiles/gun_attachables.dm:3219]: vertical?

File: code/modules/projectiles/gun_attachables.dm [hunk 2]
```diff
@@ -3241,6 +3262,9 @@ Defined in conflicts.dm of the #defines folder.
 	scatter_unwielded_mod = -SCATTER_AMOUNT_TIER_9
 	accuracy_unwielded_mod = HIT_ACCURACY_MULT_TIER_1
 
+/obj/item/attachable/lasersight/union
+	name = "Zvezda Laser Module"
+	desc = "Visible band 512nm green laser, compatible with NVG and IR. Bears striking resemblance to an old Weyland Yutani offering... Decreases scatter, especially onehanded, slightly boosts accuracy. "
```
Comment (AndroBetel) [inline code/modules/projectiles/gun_attachables.dm:3267]: green?

**Review by AndroBetel — CHANGES_REQUESTED** [review]

**Comment by AndroBetel** [comment]
upp attachies could also use some custom sprites ig... to fit the theme better... but mostly whatever...

File: code/modules/projectiles/gun_attachables.dm (hunk 2 above)
Comment (KoishiVibe) [reply code/modules/projectiles/gun_attachables.dm:3267]: aw piffle. I knew I forgor'd something. ....
I'll sprite something. Soon.

Comment (KoishiVibe) [reply code/modules/projectiles/gun_attachables.dm:3271]: sprited lol

**Review by Doubleumc — CHANGES_REQUESTED** [review]
Subtype them as `upp` instead of `union`, for example: `/obj/item/attachable/angledgrip/upp` instead of `/obj/item/attachable/angledgrip/union`

Everything else UPP is typed `upp`, these should be as well.

File: code/modules/projectiles/gun_attachables.dm [hunk 3]
```diff
@@ -3245,6 +3266,14 @@ Defined in conflicts.dm of the #defines folder.
 	scatter_unwielded_mod = -SCATTER_AMOUNT_TIER_9
 	accuracy_unwielded_mod = HIT_ACCURACY_MULT_TIER_1
 
+/obj/item/attachable/lasersight/upp
+	name = "Zvezda Laser Module"
+	desc = "Visible band 512nm green laser, compatible with NVG and IR. Bears striking resemblance to an old Weyland Yutani offering... Decreases scatter, especially onehanded, slightly boosts accuracy. "
+	icon = 'icons/obj/items/weapons/guns/attachments/under.dmi'
+	icon_state = "lasersight_union"
+	attach_icon = "lasersight_union_a"
```
Comment (Doubleumc) [inline code/modules/projectiles/gun_attachables.dm:3274]:
```suggestion
	icon_state = "lasersight_upp"
	attach_icon = "lasersight_upp_a"
```
Will want to change the icon name here and in `under.dmi` as well.

Other than that, code looks good to me.

Comment (KoishiVibe) [reply code/modules/projectiles/gun_attachables.dm]: thx for the catch yo

Comment (Doubleumc) [reply code/modules/projectiles/gun_attachables.dm]: You still need to change the icon names in `under.dmi` to match.

Comment (KoishiVibe) [reply code/modules/projectiles/gun_attachables.dm]: goddamnit-
dmc
>:(

**Review by Doubleumc — APPROVED** [review]

**Pull request merged**